src/boot/boot.js

```diff
@@ -51,9 +65,27 @@ function injectScript(doc, src) {
   // Set 'async' to false to maintain execution order of scripts.
   // See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script
   script.async = false;
+
+  tagElement(script);
   doc.head.appendChild(script);
 }
 
+/**
+ * @param {Document} doc
+ * @param {string} rel
+ * @param {'html'|'javascript'} type
+ * @param {string} url
+ */
+function injectLink(doc, rel, type, url) {
+  const link = doc.createElement('link');
+  link.rel = rel;
+  link.href = url;
+  link.type = 'application/annotator+' + type;
```

Comment:
Curious if we have a rough convention for when to use template literals or if it's up to the dev. Because of the literal `+` in this string, I'd personally find it easier to parse as

```
`application/annotator+${type}`
```

but this is quite trivial...

Reply:
> Curious if we have a rough convention for when to use template literals or if it's up to the dev.

We don't have a strong convention here. I'm happy to make this change.

Reply:
And I want to emphasize that we don't _need_ to :). Getting that micro-managey isn't something all that useful.